packages/react-component-library/package.json

```diff
@@ -170,6 +170,7 @@
     "focus-trap": "^6.9.0",
     "focus-trap-react": "^8.11.0",
     "lodash": "^4.17.21",
+    "merge-refs": "^1.3.0",
```

Comment:
This already exists I think - see `react-merge-refs` in deps.

We should just use the one already there or replace everywhere if this ones better.